src/common/packager.ts

```diff
@@ -54,7 +54,17 @@ export class Packager {
     constructor(private workspacePath: string, private projectPath: string, private packagerPort?: number, packagerStatusIndicator?: PackagerStatusIndicator) {
         this.packagerStatus = PackagerStatus.PACKAGER_STOPPED;
         this.packagerStatusIndicator = packagerStatusIndicator || new PackagerStatusIndicator();
-        this.expoHelper = new ExponentHelper(this.workspacePath, this.projectPath);
+    }
+
+    public setExponentHelper(expoHelper: ExponentHelper) {
+        this.expoHelper = expoHelper;
+    }
+
+    public getExponentHelper(): ExponentHelper{
```

Comment:
Should we generate new ExponentHelper on get? I suppose it might be better if we generate it somewhere outside and will just return the value.